aiogremlin/process/graph_traversal.py

```diff
@@ -22,25 +22,23 @@ def __aiter__(self):
         return object
 
     async def toList(self):
-        results = []
-        async for result in self:
-            results.append(result)
-        return results
+        """Return results as ``list``."""
+        return [result async for result in self]
```

Comment:
This async list comprehensions are sweet, but do they work with Python 3.5?

Reply:
I hope so 😆  We will find out when https://github.com/davebshow/aiogremlin/issues/11 is done.

Reply:
Spoiler: They don't :japanese_ogre: 

They were an enhancement introduced in Python 3.6 with [PEP 530](https://www.python.org/dev/peps/pep-0530/)